### DISCUSSION TITLE
Consiglio: prodotti ZigBee, Home Assistant e Google

### DISCUSSION THREAD

**Discussion by officialdavide** [indomus/forum#987] [GENERALE - Discussioni generiche]
Ciao, in base alle vostre esperienze, quale marca di prodotti ZigBee (inclusi hub, door sensor, ...) si interfacciano almeno a Google Assistant più Home Assistant?

Ho visto gli Aqara, molto belli, ma non si interfacciano con Google Assistant. 

Grazie ragazzi/e!

**Comment by essegi65** [comment]
Ci sono vari modelli e anche diverse tecnologie (Zigbee2mqtt, ZHA, ecc.) vai su https://zigbee.blakadder.com/index.html (spero sia possibile mettere il link) e ti farai un idea.
A riguardo l'interfaccia con Google assistant non vedo problemi una volta integrati in HA li puoi esporre e/o comandare anche da Google assistant.
